mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/monitor/IndexingMonitor.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.orm.massindexing.monitor;
```

Comment:
It does not make sense to separate the interfaces if you put them both in the "mass indexing" package. Please merge them back into a single `MassIndexingMonitor` interface, like we had agreed to. We can always extract a superinterface later, if it turns out to make sense.

Reply:
I kept both in order to preserve the v5 authoring. I'll merge them in a subsequent (not fixup) commit.

Reply:
Fair enough.